cli/command/container/opts_test.go

```diff
@@ -79,7 +79,7 @@ func TestParseRunLinks(t *testing.T) {
 		t.Fatalf("Error parsing links. Expected []string{\"a:b\", \"c:d\"}, received: %v", hostConfig.Links)
 	}
 	if _, hostConfig, _ := mustParse(t, ""); len(hostConfig.Links) != 0 {
-		t.Fatalf("Error parsing links. No link expected, received: %v", hostConfig.Links)
+		t.Fatalf("Error parsing links. No link expectedCompletions, received: %v", hostConfig.Links)
```

Comment:
I _think_ this was an accidental "find and replace"

Reply:
Ah, yes. Thanks for spotting.